indico/modules/receipts/client/js/printing/EventImageSelectField.jsx

```diff
@@ -0,0 +1,135 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Form,
+  Header,
+  HeaderContent,
+  HeaderSubheader,
+  Icon,
+  IconGroup,
+  Popup,
+} from 'semantic-ui-react';
+
+import {Translate} from 'indico/react/i18n';
+
+import './EventImageSelectField.module.scss';
+
+const imageShape = {
+  identifier: PropTypes.string.isRequired,
+  filename: PropTypes.string.isRequired,
+  preview: PropTypes.string.isRequired,
+};
+
+const imageSource = {
+  images: Translate.string('from Images'),
+  attachments: Translate.string('from Materials'),
+  placeholder: Translate.string('from Indico'),
```

Comment:
Should we show this when actually generating documents? Or maybe rename it to something like "Dummy" or "Placeholder"?

Reply:
Oh, I thought that the placeholder is currently being shown all the time. If there's no placeholder at that point it's already fine :)